### ISSUE TITLE
DMS alert: SUMO Update Contributor Metrics hasn't checked in

### ISSUE THREAD

**Issue by jgmize** [mozilla/kitsune#3647]
[SUMO Update Contributor Metrics](https://deadmanssnitch.com/snitches/42594bae2a) hasn't checked in since #3621

**Comment by jgmize** [comment]
Found the following traceback in the logs:
```
Traceback (most recent call last):
  File "/app/manage.py", line 32, in <module>
    execute_from_command_line(sys.argv)
  File "/venv/lib/python2.7/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/venv/lib/python2.7/site-packages/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/venv/lib/python2.7/site-packages/newrelic/hooks/framework_django.py", line 983, in _nr_wrapper_BaseCommand_run_from_argv_
    return wrapped(*args, **kwargs)
  File "/venv/lib/python2.7/site-packages/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/venv/lib/python2.7/site-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/venv/lib/python2.7/site-packages/newrelic/api/function_trace.py", line 127, in literal_wrapper
    return wrapped(*args, **kwargs)
  File "/app/kitsune/kpi/management/commands/process_exit_surveys.py", line 47, in handle
    emails = get_email_addresses(survey, startdatetime, enddatetime)
  File "/app/kitsune/kpi/surveygizmo_utils.py", line 85, in get_email_addresses
    total_pages = results['total_pages']
KeyError: 'total_pages'
[2019-05-23 17:00:12.464464] Clock job job_process_exit_surveys@sumo-prod-cron-79bd545fc8-vg4tw: CRASHED: Command 'python /app/manage.py process_exit_surveys' returned non-zero exit status 1
```

**Comment by jgmize** [comment]
Found the following error in the logs: `manage.py update_contributor_metrics: error: too few arguments`

The update_contributor_metrics command [needs a "day" argument](https://github.com/mozilla/kitsune/pull/3532/files#diff-dd41e518f77938dfe2e4074f6ad40f93R33) to work but it is not being passed one from `scripts/cron.py` as of https://github.com/mozilla/kitsune/pull/3532/files#diff-9d35a2d87ae415b7e8126e0570e0229eR169

**Comment by jgmize** [comment]
@SmileyChris is this something you could take a look at?

**Comment by jgmize** [comment]
Reopening this issue until the fix from #3705 is in production.
